.azure-pipelines.yml

```diff
@@ -29,193 +29,7 @@ stages:
 - stage: build_test
   displayName: Build and Test
   jobs:
-  # TODO: Consider running the Docker job only if the Dockerfile was changed.
-  - job: 'Docker'
-    pool:
-      vmImage: 'ubuntu-18.04'
-    # TODO: Add caching.
-    steps:
-    - task: Docker@2
-      inputs:
-        command: 'build'
-        Dockerfile: 'Dockerfile'
-        tags: |
-          ort
-
+  - template: ci/docker.yml
```

Comment:
How about doing this commit first to introduce the `ci` sub-directory, and *then* also use that structure for the new `detekt` job?

Reply:
Well that's the order I implemented this, I dont' see any benefit in changing it. I first tested templates for the new job, then migrated the others when I saw it worked.

Reply:
The benefit is to avoid confusion when reviewing commit by commit, as I was wondering why you create a sub-directory for only a single file, because the template approach could have been tested also without that sub-directory. Only in the next commit, as more jobs get "templatized", the sub-directory really makes sense. That's why from a reviewer's perspective the other order makes more sense IMO, even if you tried it out in this order.

Anyway, that's just a remark from my side.